app/models/user.py

```diff
@@ -53,9 +77,19 @@ class User(UserMixin, db.Model):
     email = db.Column(db.String(64), unique=True, index=True)
     password_hash = db.Column(db.String(128))
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
+
+    donor_level_id = db.Column(db.Integer, db.ForeignKey('donorlevels.id'))
+    age = db.Column(db.Integer)
```

Comment:
Would it be preferable to change this field to a birthday so we can dynamically track someone's age over time?